app/assets/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,153 @@
+.content {
```

Comment:
Sachi, it is very cool that you managed to make this animation work! However, in this scenario there is a solution that was offered to us by Le Wagon to do something very similar. It is called Typed JS (https://github.com/mattboldt/typed.js/). It is a more elegant solution for the same issue. How do you feel about trying it out tomorrow? 😉